### ISSUE TITLE
Elemental - Remove unused sub-menus

### ISSUE THREAD

**Issue by juadk** [rancher/dashboard#6894]
**Setup**
- Rancher version: 2.6.7
- Rancher-dashboard branch: elemental-dev

**Describe the bug**
That's not really a bug, actually I don´t understand why we have these two sub-menu in the Elemental UI:
- Mach. Inv. Selec. Templates
- Mach. Inv. Selectors
![image](https://user-images.githubusercontent.com/6025636/190190065-430fbe94-340e-438a-a77c-c3bca822e3cd.png)

Because from my understanding, these things are managed in the cluster screen from the Rancher's screen:
![image](https://user-images.githubusercontent.com/6025636/190190302-2d117e49-2e5e-4314-b793-98632ce71277.png)

I would remove them from the elemental UI menu but final words are for Elemental devs and UI folks.

**Comment by ldevulder** [comment]
Yes I agree that these menus are not really useful IMO. For example the `MachineInventorySelectorTemplate` can be defined directly when the cluster is created (as @juadk said) but the `MachineInventorySelector` entries are then created automatically from the `Template`. So not really useful to create them manually, and it can be denagerous to edit them in fact.

**Comment by kkaempf** [comment]
I tend to agree as, to my understanding, selectors and their templates are for cluster creation (generic for any type of machine or cluster) and not for OS management.

**Comment by kwwii** [comment]
We need to look into this from a UX perspective and understand the details as this is directly associated with the cluster provisioning workflow

**Comment by agracey** [comment]
That's likely too large of a change for now, just a thought.

**Comment by agracey** [comment]
For now, we should definitely hide the selector that gets created from the user and only expose the parts they need to interact with
